spec/services/calculators/disqualification_calculator_spec.rb

```diff
@@ -19,26 +19,20 @@
     let(:under_age) { GenericYesNo::YES }
 
     describe "#expiry_date" do
-      context "with a lengthless than or equal to 2.5 years" do
+      context "with a length less than or equal to 2.5 years" do
```

Comment:
These contexts don't seem relevant anymore as the calculation doesn't change based on under or over a certain length.